Kickstarter-iOS/Views/Cells/PledgeShippingLocationCell.swift

```diff
@@ -3,9 +3,14 @@ import Library
 import Prelude
 import UIKit
 
+protocol PledgeShippingLocationCellDelegate: AnyObject {
+  func pledgeShippingCell(_ cell: PledgeShippingLocationCell, didUpdateShippingAmount amount: Double)
```

Comment:
I wonder if we should just pass back the whole `ShippingRule` which includes `shippingRule.cost` (`Double`) but also the shipping location. I know we'll definitely need to send the shipping location to the `createBacking` mutation, so we might as well pass back the whole thing now? 🤷‍♀

Reply:
Sure, I can update that.